src/tools/meson.build

```diff
@@ -1,13 +1,13 @@
-executable('6040macal', '6040macal.c', dependencies: [wlmio_dep], install: true)
-executable('6050cal', '6050cal.c', dependencies: [wlmio_dep], install: true)
-executable('6070cal', '6070cal.c', dependencies: [wlmio_dep], install: true)
-executable('6090cal', '6090cal.c', dependencies: [wlmio_dep], install: true)
-executable('6090ccal', '6090ccal.c', dependencies: [wlmio_dep], install: true)
-executable('6180a_cal', '6180a_cal.c', dependencies: [wlmio_dep], install: true)
-executable('6190a_conf', '6190a_conf.c', dependencies: [wlmio_dep], install: true)
-executable('factoryreset', 'factoryreset.c', dependencies: [wlmio_dep], install: true)
-executable('fwupdate', 'fwupdate.c', dependencies: [canard_dep, libgpiod_dep], install: true)
-executable('infodump', 'infodump.c', dependencies: [wlmio_dep], install: true)
-executable('monitor', 'monitor.c', dependencies: [wlmio_dep], install: true)
-executable('regtool', 'regtool.c', dependencies: [wlmio_dep], install: true)
-executable('store', 'store.c', dependencies: [wlmio_dep], install: true)
+executable('wl_6040macal', '6040macal.c', dependencies: [wlmio_dep], install: true)
+executable('wl_6050cal', '6050cal.c', dependencies: [wlmio_dep], install: true)
+executable('wl_6070cal', '6070cal.c', dependencies: [wlmio_dep], install: true)
+executable('wl_6090cal', '6090cal.c', dependencies: [wlmio_dep], install: true)
+executable('wl_6090ccal', '6090ccal.c', dependencies: [wlmio_dep], install: true)
+executable('wl_6180a_cal', '6180a_cal.c', dependencies: [wlmio_dep], install: true)
+executable('wl_6190a_conf', '6190a_conf.c', dependencies: [wlmio_dep], install: true)
+executable('wl_factoryreset', 'factoryreset.c', dependencies: [wlmio_dep], install: true)
+executable('wl_fwupdate', 'fwupdate.c', dependencies: [canard_dep, libgpiod_dep], install: true)
+executable('wl_infodump', 'infodump.c', dependencies: [wlmio_dep], install: true)
+executable('wl_monitor', 'monitor.c', dependencies: [wlmio_dep], install: true)
+executable('wl_regtool', 'regtool.c', dependencies: [wlmio_dep], install: true)
+executable('wl_store', 'store.c', dependencies: [wlmio_dep], install: true)
```

Comment:
These names specifically are problematic in their generic nature the better alternative might be changing the prefix to `/usr/libexec/widgetlords/` or `/opt/widgetlords/`

Reply:
Have you seen any conflicts in the wild?